index.js

```diff
@@ -1,8 +1,6 @@
 "use strict";
 
-const {CompositeDisposable, Disposable} = "object" === typeof atom
-	? require("atom")
-	: require("event-kit");
+const {CompositeDisposable, Disposable} = require("atom")   // use `require("event-kit");` in non-atom env.
```

Comment:
Remove the comment and add a semicolon, please.

Reply:
Fixed in https://github.com/file-icons/mapped-disposable/pull/2/commits/69a459724f8fea596ffed256da224dfbb561fe4d